modules/ROOT/pages/7.1-release-notes.adoc

```diff
@@ -151,29 +288,154 @@ For information on using Enhanced Skins & Icon Packs, see: xref:enhanced-skins-a
 [[bug-fixes]]
 == Bug fixes
 
-{productname} 7.1 also includes the following bug fix<es>:
+{productname} {release-version} also includes the following bug fixes:
 
-=== <TINY-vwxyz 1 changelog entry>
-// #TINY-vwxyz1
+=== Video and audio elements could not be played on Safari.
```

Comment:
```suggestion

=== Improved the responsiveness of the custom view on small screens
// #TINY-10741

Previously, when opening a custom view on small screens or mobile devices, like the Revision History view, some header buttons and UI components were not completely visible, and in some cases, they were not accessible. For example, users might not have been able to close the view because the Close button was obscured on smaller screens.

{productname} {release-version} addresses this issue by improving the layout's responsiveness to maximize usability.

As a result, header buttons and UI components are more accessible and visible on small screens and mobile devices, ensuring that users can interact with the custom view without issues.

For more information, see the xref:custom-view.adoc[Custom views] documentation.

=== Video and audio elements could not be played on Safari.
```